### PR TITLE
Improved error message when a gem is not installed

### PR DESCRIPTION
**Pull request by KrauseFx**
Fixes https://github.com/fastlane/fastlane/issues/1339

### CODE REVIEW

File: lib/fastlane/helper/gem_helper.rb [hunk 1]
```diff
@@ -8,22 +8,19 @@ def self.verify_gem!(gem_name)
...
+        UI.error("If you installed fastlane using `sudo gem install fastlane` run")
+        UI.command("sudo gem install #{gem_name}")
+        UI.error("to install the missing gem")
+        UI.error("")
+        UI.error("If you use a Gemfile add this to your Gemfile:")
+        UI.important("  gem '#{gem_name}'")
+        UI.error("and run `bundle install`")
 
-        raise "You have to install the `#{gem_name}`".red unless Helper.is_test?
+        UI.user_error!("You have to install the `#{gem_name}` gem on this maching") unless Helper.is_test?
```
Comment (mfurtak) [inline lib/fastlane/helper/gem_helper.rb]: 'maching' -> 'machine'

**Comment by KrauseFx** [comment]
Updated :+1:

**Comment by mfurtak** [comment]
:cool: :ship: it

**Comment by KrauseFx** [comment]
Thanks :+1:

**Pull request merged**